doc/tasks/php_parser.md

```diff
@@ -0,0 +1,73 @@
+# Phpparser
+
+The Phpparser task will test your changes for blacklisted or whitelisted keywords, such as `die()`, `var_dump()` etc.
+It lives under the `php_parser` namespace and has following configurable parameters:
+
+```yaml
+# grumphp.yml
+parameters:
+    tasks:
+        php_parser:
+            visitors_options:
+              function_call_visitor:
+                blacklist: ["var_dump", "error_log"]
+                whitelist: ["count", "get_class"]
+              concrete_method_call_visitor:
+                blacklist: ["print_r", "run"]
+              static_method_call_visitor:
+                blacklist: ["var_export"]
+            visitors:
+              - {visitor: '@grumphp.parser.php.visitor.function_call'}
+              - {visitor: '@grumphp.parser.php.visitor.concrete_method_call'}
+              - {visitor: '@grumphp.parser.php.visitor.static_method_call'}
+            triggered_by: [php]
+            ignore_patterns: []
+```
+
+**visitors_options**
```

Comment:
Each class visitor as it's own configuration:

When class named `Namespace\To\ClassFooVisitor` the option entry should underscore named as base class name. In this example it will be `class_foo_visitor`